src/Main.elm

```diff
@@ -1,10 +1,362 @@
 module Main exposing (main)
 
-import Data.Audience
-import Data.AudienceFolder
-import Html exposing (Html)
+import Browser
+import Data.Audience as Audience
+import Data.AudienceFolder as AudienceFolder
+import Decoder
+import Html exposing (Html, button, div, text, ul)
+import Html.Attributes exposing (class)
+import Html.Events exposing (onClick)
+import Http exposing (Error(..))
 
 
-main : Html msg
+type Msg
+    = MsgGotResultsAudienceFolder (Result Http.Error (List AudienceFolder.AudienceFolder))
+    | MsgGotResultsAudience (Result Http.Error (List Audience.Audience))
+    | OpenFolder Int
+    | GoBack
+    | SelectCategory Audience.AudienceType
```

Comment:
This `Msg` is fired to handle the filters, but they are done by `AudienceType`s, not by _categories_ (which are not used here).
```suggestion
    | SelectAudienceType Audience.AudienceType
```